networkx/algorithms/tests/test_covering.py

```diff
@@ -0,0 +1,82 @@
+from nose.tools import assert_equal, assert_true, assert_false
+import networkx as nx
+
+
+class TestMinEdgeCover:
+    """Tests for :func:`networkx.algorithms.min_edge_cover`"""
+
+    def setUp(self):
+        pass
+
+    def test_empty_graph(self):
+        G = nx.Graph()
+        assert_equal(nx.min_edge_cover(G), set())
+
+    def test_graph_with_loop(self):
+        G = nx.Graph()
+        G.add_edge(0, 0)
+        assert_equal(nx.min_edge_cover(G), {(0, 0)})
+
+    def test_graph_single_edge(self):
+        G = nx.Graph()
+        G.add_edge(0, 1)
+        assert_equal(nx.min_edge_cover(G),
+                     {(0, 1), (1, 0)})
+
+    def test_bipartite(self):
```

Comment:
Why are you testing bipartite matchings explicitly here? These tests are not for the bipartite-specific function.